### ISSUE TITLE
[BUG] -Installation issue

### ISSUE THREAD

**Issue by SmkyMtn** [invernyx/smartcars-3-bugs#124]
![smartcars3](https://user-images.githubusercontent.com/131419863/233523853-6b0d455f-ba93-4072-99f5-90ac0f016acf.JPG)
**Describe the bug**

**To Reproduce**
Get Java Error everytime I try to run the program, this is the first time trying to run it after multiple installs. 

**Expected behavior**
To run without giving an error

**Screenshots**
attached as a file

**System Information (please complete the following information):**
 - OS: Windows 10
 - Community ID: 
 - Version 
 - Plugins used: 

**Additional context**

**Comment by GenericNerd** [comment]
This is a strange issue to come across. I had this when I tried to run smartCARS on Linux. Can you try the following steps:

1. Uninstall smartCARS 3
2. Remove all files from %localappdada%\TFDi Design
3. Reinstall smartCARS

If this issue keeps happening, let us know

**Comment by HondaJet** [comment]
I followed the steps suggested above for my Java error.  After steps 1-3 were done it still gave me the Java error when I started SC3.  I have always used start sim programs "as admin" and my shortcut was set up that way.   However after all this I tried it with the as admin box unchecked and now SC3 is opening as appropriate.  I did not find anything in the manual about NOT using "as admin" to start the program.   Maybe other people will find this helpful
Jerry
